install/data/tyk.with_dash.conf

```diff
@@ -49,7 +49,7 @@
     "analytics_config": {
         "type": "",
         "ignored_ips": [],
-        "enable_detailed_recording": true,
+        "enable_detailed_recording": false,
```

Comment:
Changed this for performance, do we want to enable this by default?

Reply:
Makes sense